src/libpriv/rpmostree-importer.cxx

```diff
@@ -182,14 +182,18 @@ build_rpmfi_overrides (RpmOstreeImporter *self)
   const gboolean doc_files_are_filtered = (*self->importer_rs)->doc_files_are_filtered ();
   while ((i = rpmfiNext (self->fi)) >= 0)
     {
+      if (g_cancellable_set_error_if_cancelled (cancellable, error))
+        return FALSE;
       const char *user = rpmfiFUser (self->fi);
       const char *group = rpmfiFGroup (self->fi);
       const char *fcaps = rpmfiFCaps (self->fi);
       const char *abs_filepath = rpmfiFN (self->fi);
-      g_assert (abs_filepath != NULL);
-      g_assert (abs_filepath[0] == '/');
-      rpmfileAttrs fattrs = rpmfiFFlags (self->fi);
+      if (abs_filepath == NULL)
+        return glnx_throw (error, "Missing absolute filepath");
```

Comment:
Minor nit but this error message seems confusing...I'd say more like "Missing expected filename" or something?  It's not clear to me it could actually happen, but yeah probably better to error anyways.